packages/mural-account-chooser/src/components/account-chooser/sign-up-with-3rd-party.tsx

```diff
@@ -0,0 +1,137 @@
+  background: ${({ theme }) => theme.contentBackgroundColor};
+`;
+const Header = styled.h1`
+  margin: 20px;
+  font-family: ${FONT_FAMILY};
+  line-height: 100%;
+  text-align: center;
+  font-size: 2em;
+  font-weight: bold;
+`;
+const ThirdPartySignUpButton = styled.button`
+  cursor: pointer;
+  width: 70%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-size: 1em;
+  font-weight: bold;
+  font-family: ${FONT_FAMILY};
+  color: ${({ theme }) => theme.primaryTextColor};
+
+  background: ${({ theme }) => theme.contentBackgroundColor};
+  border: 1px solid;
+  border-radius: 10px;
+  padding: 10px;
+`;
+const Icon = styled.img`
+  height: 32px;
+`;
+const Text = styled.div`
+  font-family: ${FONT_FAMILY};
+  color: ${({ theme }) => theme.secondaryTextColor};
+  margin: 10px;
+`;
+const Separator = styled.div`
+  /* Size / Position */
+  position: relative;
+  width: 260px;
+  margin: 20px;
+
+  /* Other */
+  text-align: center;
+  font-family: ${FONT_FAMILY};
+  color: ${({ theme }) => theme.secondaryTextColor};
+
+  &:before {
+    content: '';
+    display: inline-block;
+    width: 50%;
+    margin: 0 0.5em 0 -55%;
+    vertical-align: middle;
+    border-bottom: 1px solid;
+  }
+
+  &:after {
+    content: '';
+    display: inline-block;
+    width: 50%;
+    margin: 0 0.5em 0 -55%;
+    vertical-align: middle;
+    border-bottom: 1px solid;
+    margin: 0 -55% 0 0.5em;
+  }
+`;
+const SendVerificationEmail = styled.button`
+  /* Button */
+  cursor: pointer;
+  background: none;
+  padding: 0;
+  margin: 10px;
+  border: none;
+  border-bottom: 1px dashed ${({ theme }) => theme.secondaryTextColor};
+
+  /* Font */
+  font-family: ${FONT_FAMILY};
+  color: ${({ theme }) => theme.primaryTextColor};
+  font-weight: bold;
+  font-size: 0.9em;
+  line-height: 120%;
+`;
+
+interface PropTypes {
+  name: string;
+  iconSrc?: string;
+  signUp: MouseEventHandler;
+  sendVerificationEmail?: MouseEventHandler;
+  theme?: {
+    primaryTextColor: string;
+    secondaryTextColor: string;
+    backgroundColor: string;
+    contentBackgroundColor: string;
+  };
+}
+
+const SignUpWith3rdParty: React.FC<PropTypes> = (props: PropTypes) => {
+  const { name, iconSrc, signUp, sendVerificationEmail, theme } = props;
+  const icon = iconSrc ?? MuralIcon;
+
+  return (
+    <ThirdPartySignUp theme={theme}>
+      <Header>Let's verify your email</Header>
+      <ThirdPartySignUpButton
+        onClick={signUp}
+        data-qa="sign-up-with"
+        theme={theme}
+      >
+        <Icon src={icon} alt="avatar" />
+        <Text>Continue with {name}</Text>
+      </ThirdPartySignUpButton>
+
+      <Separator>or</Separator>
+      <SendVerificationEmail
+        onClick={sendVerificationEmail}
+        data-qa="send-verification-email"
+        theme={theme}
+      >
+        Send me a verification email
+      </SendVerificationEmail>
+    </ThirdPartySignUp>
+  );
+};
+
+export default SignUpWith3rdParty;
```

Comment:
**notes**: I have some comments about the layout portion of this CSS as there are quite a bit of magic numbers thrown around, but I won't block this PR because 1) we don't really have a documented approach to do this 2) this should be deal with using the design system foundation CSS baseline 3) we can fix this later.

Just for posterity's sake, and also because I'll bring this up to the team, here are the basic rules I would like us to follow:

1. Use `em` for typographic content sizing (font-size, line-height) 
2. Use `rem` for layouting (margin, padding, left, top, gaps)
3. Use `px` only for pixel-perfect elements (images, icons, glyphs, borders)
4. Use `flex` rules for containers (width, height)